docs/user-guide/release-notes/2.14.x.rst

```diff
@@ -10,3 +10,6 @@ New Features
 
 * New parameter for rsync distributor ``rsync_extra_args`` allows user to 
   specify custom arguments for rsync call during publish operations.
+
+* New publish step RemoveOldRepodataStep removes expired repodata at the end 
```

Comment:
Please add a sentence that says `Enable this step with these new distributor options.` and have the 'these new distributor options' part link to your new settings in the docs please.